discovery-provider/src/queries/query_helpers.py

```diff
@@ -815,6 +815,58 @@ def get_repost_counts(
     return repost_counts_query.all()
 
 
+def get_karma(session, track_ids, time=None):
```

Comment:
This whole query is weirdly readable

Reply:
nice! needed u for the alley-oop on the first round :)